src/wp-admin/includes/file.php

```diff
@@ -1980,6 +1980,10 @@ function move_dir( $from, $to, $overwrite = false ) {
 		);
 	}
 
+	if ( trailingslashit( $from ) === trailingslashit( $to ) ) {
+		return false;
```

Comment:
This should be a `WP_Error` object as `move_dir()` has a return type of `true|WP_Error`.

Reply:
I also changed the return type. Was this incorrect?

Reply:
Updated to return WP_Error

Reply:
Sorry, been away from the PC a bit. Thanks for implementing the `WP_Error` :slightly_smiling_face:, this way `move_dir()` remains consistent with `copy_dir()` - It should never fail without an explanation via `WP_Error`